tests/queries/0_stateless/00804_test_alter_compression_codecs.sql

```diff
@@ -25,15 +25,23 @@ INSERT INTO alter_compression_codec VALUES('2018-01-01', 5, '5');
 INSERT INTO alter_compression_codec VALUES('2018-01-01', 6, '6');
 SELECT * FROM alter_compression_codec ORDER BY id;
 
+SET enable_qpl_deflate = 1;
```

Comment:
Could we move the re-compression part (l. 28-34) down to the other statements doing re-compression, i.e. before l. 39?

Reply:
done